blocks/bricks/__init__.py

```diff
@@ -533,6 +605,108 @@ def categorical_cross_entropy(self, y, x):
         return cost
 
 
+class LinearActivationDocumentation(_Brick):
+    """Dynamically adds documentation to LinearActivation descendants.
+
+    Notes
+    -----
+    See http://bugs.python.org/issue12773.
+
+    """
+    def __new__(cls, name, bases, classdict):
+        classdict['__doc__'] = \
+            """Linear transformation followed by elementwise application of \
+            {0} function.""".format(name[6:].lower())
+        if 'apply' in classdict:
```

Comment:
This should not happen - similarly to the `ActivationDocumentation` class the `LinearActivationDocumentation` is supposed to only work with `LinearActivation` which provides an `apply` method. But a check will not hurt.